torch/csrc/jit/ir/named_value.h

```diff
@@ -60,6 +54,10 @@ struct NamedValue {
     return value_;
   }
 
+  bool hasName() const {
+    return name_.has_value();
+  }
```

Comment:
Where is this new method used?

Reply:
More codebase cleanup. We throw if `name_` doesn't exist when we call `name()`, but we have no way to check if `name_` actually does exist beforehand to prevent this error. I'll delete this